app/controllers/application_controller.rb

```diff
@@ -62,4 +62,8 @@ def render_unauthorized
     headers['WWW-Authenticate'] = 'Token realm="Application"'
     render json: 'Not Authorized', status: 401
   end
+
+  rescue_from ActionController::ParameterMissing do
+    render json: { error: 'parameter missing' }, status: :bad_request
+  end
```

Comment:
There is a custom exception in VA::API::Common for handling Missing Parameter if you want to handle that case and raise the exception, otherwise we can revisit later. The custom exception will give you the proper JSON API spec for errors though.